examples/collect/host/run-and-save-output.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: troubleshoot.sh/v1beta2
+kind: HostCollector
+metadata:
+  name: run-host-cmd-and-save-output
+spec:
+  collectors:
+    - run:
+        collectorName: "my-custom-run"
+        command: "cat $TS_INPUT_DIR/dummy.yaml"
```

Comment:
This will also not work. It needs to be something like below so that the environmen

```yaml
        command: "cat"
        args: [ "$TS_INPUT_DIR/dummy.yaml" ]
```

Reply:
done